src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public abstract class Figure implements AreaCalculator, Drawable {
+    private String color;
+
+    public Figure(String color) {
+        this.color = color;
+    }
+
+    public String getColor() {
+        return color;
+    }
```

Comment:
access it directly in child classes. color not accessed from outside, so can be protected without getters
```suggestion
    protected String color;

    public Figure(String color) {
        this.color = color;
    }
```